libs/form-builder/src/components/FormBuilder.tsx

```diff
@@ -0,0 +1,69 @@
+import { FormProvider, useForm } from 'react-hook-form';
+
+import { FormLayout } from '@daohaus/ui';
+
+import { FormBuilderFactory } from './FormBuilderFactory';
+import { FormLego } from '../types/legoTypes';
+
+import { Logger } from './Logger';
+import { FormFooter } from './formFooter';
+import { useState } from 'react';
+import { isValidNetwork } from '@daohaus/common-utilities';
+import { useHausConnect } from '@daohaus/daohaus-connect-feature';
+
+type FormBuilderProps = FormLego & {
+  onSubmit: (
+    formValues: Record<string, unknown>
+  ) => void | Promise<(formValues: Record<string, unknown>) => void>;
+  onCancel?: () => void;
+  onSuccess?: () => void;
+  onError?: () => void;
+};
+
+export const FormBuilder = ({
+  title,
+  subtitle,
+  description,
+  fields,
+  onSubmit,
+  log,
+}: FormBuilderProps) => {
+  const methods = useForm();
+  const {
+    formState: { isValid },
+  } = methods;
+  const { chainId } = useHausConnect();
```

Comment:
Yeah, was thinking builders would be super high-level for now. But maybe I could make this props? 

It would be awesome if we could find a way to use connect or anything else. 

I'd be down if we made a ticket to remove these deps down the road.